modules/claims_api/app/workers/claims_api/ews_updater.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'bgs'
+require 'claims_api/claim_logger'
+
+module ClaimsApi
+  class EwsUpdater
+    include Sidekiq::Worker
+
+    def perform(bgs_claim, target_veteran)
+      bgs_service = BGS::Services.new(
+        external_uid: target_veteran[:external_uid],
+        external_key: target_veteran[:external_key]
+      )
+      response = bgs_service.benefit_claims.update_bnft_claim(claim: bgs_claim)
+
+      if !response[:bnft_claim_dto].nil?
+        bgs_claim[:status] = ClaimsApi::EvidenceWaiverSubmission::UPDATED
```

Comment:
this would be the claim we received from BGS. I believe we'll want to update our table.